frontend/messages/en.json

```
@@ -2797,6 +2797,59 @@
+	"people": "People",
+	"lifecycle": "Lifecycle",
+	"comingSoon": "Coming soon",
+	"quality": "Quality",
+	"benefits": "Benefits",
+	"subProjects": "Sub-projects",
+	"noSubProjects": "No sub-projects",
+	"noResponsibilityMatrices": "No responsibility matrices",
+	"financials": "Financials",
+	"expectedBudget": "Expected budget",
+	"actualCost": "Actual cost",
+	"remaining": "Remaining",
```

Biome (2.4.15)

[error] The key remaining was already declared.

(lint/suspicious/noDuplicateObjectKeys)